client/state/data-layer/analytics/adTracking.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { recordSelectedVertical } from 'lib/analytics/ad-tracking';
+import { SIGNUP_STEPS_SURVEY_SET } from 'state/action-types';
+
+function handleSignupStepsSurveyStart( _, action, next ) {
```

Comment:
that sounds good, plus I'd rather have two similarly named functions which indicate what they do than one that does and one which doesn't